packaging/suse/make_spec.sh

```diff
@@ -36,11 +46,11 @@ additional_native_build_requirements() {
   fi
 }
 
-mkdir -p build/Portus-$branch
```

Comment:
@jordimassaguerpla do you have an idea if the capitalization is important here? the new version will render `build/portus-master` instead of `build/Portus-master`